x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -6,40 +6,51 @@
  */
 
 import React from 'react';
-import { EuiSpacer } from '@elastic/eui';
+import { EuiSpacer, EuiIcon } from '@elastic/eui';
 import { allNavigationItems } from '../../common/navigation/constants';
 import { useCspBreadcrumbs } from '../../common/navigation/use_csp_breadcrumbs';
 import { SummarySection } from './dashboard_sections/summary_section';
 import { BenchmarksSection } from './dashboard_sections/benchmarks_section';
-import { useCloudPostureStatsApi } from '../../common/api';
-import { CspPageTemplate } from '../../components/page_template';
-import * as TEXT from './translations';
+import { useComplianceDashboardDataApi } from '../../common/api';
+import { CspPageTemplate } from '../../components/csp_page_template';
+import { type KibanaPageTemplateProps } from '../../../../../../src/plugins/kibana_react/public';
+import { CLOUD_POSTURE, NO_DATA_CONFIG_TEXT } from './translations';
 
-const CompliancePage = () => {
-  const getStats = useCloudPostureStatsApi();
-  if (getStats.isLoading) return null;
-
-  return (
-    <>
-      <SummarySection />
-      <EuiSpacer />
-      <BenchmarksSection />
-      <EuiSpacer />
-    </>
-  );
-};
+const getNoDataConfig = (onClick: () => void): KibanaPageTemplateProps['noDataConfig'] => ({
```

Comment:
why does the dashboard need its own `noDataConfig` ?

Reply:
in general, I would say it's best if all pages pass their own `noDataConfig`. having no data is pretty case-specific and every page can show different title or button options. in this case, i've added a refetch button in case the data is still on its way to ES